src/www/ui/api/Controllers/LicenseController.php

```diff
@@ -134,6 +134,43 @@ public function getLicense($request, $response, $args)
     return $response->withJson($returnVal->getArray(), 200);
   }
 
+  /**
+   * Delete the license information based on the provided parameters
+   *
+   * @param Request $request
+   * @param ResponseHelper $response
+   * @param array $args
+   * @return ResponseHelper
+   */
+  public function deleteLicense($request, $response, $args)
+  {
+    $shortName = $args["shortname"];
+
+    if (empty($shortName)) {
+      $error = new Info(
+        400,
+        "Short name missing from request.",
+        InfoType::ERROR
+      );
+      return $response->withJson($error->getArray(), $error->getCode());
+    }
+
+    $result = $this->licenseDao->deleteLicenseByShortName($shortName,
+      $this->restHelper->getGroupId(), $this->restHelper->getUserId);
+
+    if ($result === false) {
```

Comment:
As suggested about, `$result` might also be `null` and it'll be considered as deleted (even though it is not).